lib/ansible/modules/monitoring/manageiq/manageiq_user.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/python
+#
+# (c) 2017, Daniel Korn <korndaniel1@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+
+module: manageiq_user
+
+short_description: Management of users in ManageIQ
+extends_documentation_fragment: manageiq
+version_added: '2.4'
+author: Daniel Korn (@dkorn)
+description:
+  - The manageiq_user module supports adding, updating and deleting users in ManageIQ.
+
+options:
+  name:
+    description:
+      - The unique userid in manageiq, often mentioned as username
+    required: true
+    default: null
+  fullname:
+    description:
+      - The users' full name
+    required: false
+    default: null
+  password:
+    description:
+      - The users' password
+    required: false
+    default: null
+  group:
+    description:
+      - The name of the group to which the user belongs
+    required: false
+    default: null
+  email:
+    description:
+      - The users' E-mail address
+    required: false
+    default: null
+  state:
+    description:
+      - The state of the user
+      - On present, it will create the user if it does not exist or update the
```

Comment:
Use [formatting options](http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#formatting-options): for example `C(present)` instead of `present`.